junifer/api/res/run_venv.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
hard coded shell? I suppose this will change in a separate PR?

Reply:
I think this is the case. We need to provide `run_venv.zsh`, `run_conda.zsh`, etc.

Reply:
Yes this will change in #273 which will require a new option in the YAML.